### DISCUSSION TITLE
Add optional arguments after `docker run` command

### DISCUSSION THREAD

**Discussion by wajdijurry** [spatie/docker#36] [Ideas]
Needs:
When I need to pass `-it` for example to the `docker run` command, I can't since there is no option to pass these optional arguments.

Example: 
```
docker run -it --name example -d redis:latest
```

Is there any intention to add the ability to pass optional arguments to `docker run` command ?

**Comment by freekmurze** [comment]
That is currently not supported, but feel free to create a PR that adds this functionality. Make sure to update the readme and tests.

**Comment by wajdijurry** [comment]
Thanks for your quick reply. Let me create a PR.

**Comment by wajdijurry** [comment]
PR link:
https://github.com/spatie/docker/pull/37
